src/nss-mymachines/meson.build

```diff
@@ -0,0 +1,13 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+if conf.get('ENABLE_NSS_MYMACHINES') != 1
+        subdir_done()
+endif
```

Comment:
… and here too, why not `conditions : ['ENABLE_NSS_MYMACHINES']` in the definition? That way the rest of the file is "executed" even if something is skipped in the local configuration, so it's less likely to break in other configurations.